README.md

```diff
@@ -136,6 +138,14 @@ Additional attributes not defined in the [Avro Specification](https://avro.apach
 
 - `example` - Can be used to define the example value from the business domain of given field. Value will be propagated into [examples attribute](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.9.5) of JSON schema and therefore will be picked for the generated "Example of payload" when using some AsyncAPI documentation generator.
 
+### Support for names and namespaces
+
+If, at the top level of the Avro schema, the 'name' attribute is defined, it will be copied to the corresponding JSON schema's 'x-parser-schema-id' attribute. If the Avro schema also has the 'namespace' attribute defined, then that schema's fully qualified name will be put into that attribute. The fully qualified name is defined by the namespace, followed by a dot, followed by the name.
+
+If there are two schemas that resolve to the same fully qualified name, only the second schema will be returned by the parser, the first will be discarded.
```

Comment:
I have to say this one doesn't let me sleep 😄 
what part of the code is responsible for it? why 2nd is returned, then what if 3 schemas resolve to the same?

Reply:
I assume that parser-js keeps a map where the key is x-parser-schema-id. So if there are several schemas that share the same x-parser-schema-id, then only the last one parsed will end up in that map. But if their document has several schemas with the same fully qualified name, I'd hope that they're identical, they repeat them because we force them to put Avro schemas into the message/payload section instead of components/schemas.

Maybe I should just not mention that at all? Or maybe that's an argument for using the title field after all, and adding x-namespace or something?

Reply:
ah, now it makes sense, you are referring to 👇🏼 

```js
  /**
   * @returns {Map<string, Schema>}
   */
  allSchemas() {
    const schemas = new Map();
    const allSchemasCallback = (schema) => {
      if (schema.uid()) {
        schemas.set(schema.uid(), schema);
      }
    };
    traverseAsyncApiDocument(this, allSchemasCallback);
    return schemas;
  }
```

I think the sentence is ok, but just maybe write
`If there are two schemas that resolve to the same fully qualified name, only the last one will be returned by the parser. Make sure names of your schemas are unique` ?